src/ducks/payments/AccessToken.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from 'react'
+import palette from 'cozy-ui/transpiled/react/palette'
+import { LinearProgress } from 'cozy-ui/transpiled/react/Progress'
+import { styles } from './Payments'
+import { getAccessTokenPayload } from './api'
+import Banner from 'cozy-ui/transpiled/react/Banner'
+import Label from 'cozy-ui/transpiled/react/Label'
+import { useClient } from 'cozy-client'
+import { createBiPayment } from './service'
+import { usePaymentContext } from './PaymentContext'
```

Comment:
nit: Just to note that we're slowly working on improving our import statements. 

first: 
- react
- third party lib 
- cozy-lib (ui/client/ ...)
- local import 

Also I think you should import from relative path since we can't override them easily. 
Instead of 
`import { usePaymentContext } from './PaymentContext'``
write
`import { usePaymentContext } from 'ducks/payments/PaymentContext'`

Reply:
The eslint plugin for imports could be useful to activate in CCA.